Mjml.Net.Benchmark/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using BenchmarkDotNet.Running;
+using CommandLine;
+
+namespace Mjml.Net.Benchmarking
+{
+    public static class Program
+    {
+        class Options
+        {
+            [Option('p', "profiler", Required = false, HelpText = "Runs the test runenr logic.")]
+            public bool TestRunner { get; set; }
+        }
+
+        public static void Main(string[] args)
+        {
+            Parser.Default.ParseArguments<Options>(args)
+                     .WithParsed(o =>
+                     {
+                         if (o.TestRunner)
+                         {
+                             TestRunner.Run();
```

Comment:
Where is the TestRunner coming from?

Reply:
Btw: Can we also use the dynamic approach in the TestRunner?

Reply:
It's static class `TestRunner` which I copied the code from TestRunner project. https://github.com/SebastianStehle/mjml-net/blob/Additional-Benchmark-Templates/Mjml.Net.Benchmark/TestRunner.cs

Reply:
> Btw: Can we also use the dynamic approach in the TestRunner?

I'll do that now :)